recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+do_install:append:rpi() {
```

Comment:
What is the behaviour without a keyboard and without this option? This sounds like a oe-core change to me unless I'm missing something.

Reply:
Currently, if you do not attached keyboard and mouse then weston will not start on rpi.
this is only needed for SOCs using DRM backends, so its really not something we can enumerate in OE-core.
there is `DEFAULTBACKEND` which can be set and then used to deduce if a machine is using drm backend but its not a strict requirement to mention one, since weston will pick one if not mentioned. So we do it here because with rpi we are sure of which backend is in use.

I have done something similar for meta-ti

Reply:
I was thinking of DEFAULTBACKEND indeed. But it makes sense now.